tokens/_aliases.yml

```diff
@@ -95,33 +95,11 @@ aliases:
   spacing-md: '16px'
   spacing-lg: '24px'
   spacing-xl: '32px'
-  spacing-2xl: '48px'
```

Comment:
This is used pretty commonly in bedrock on page specific (i.e. non-protocol) components: https://github.com/mozilla/bedrock/search?q=%24spacing-2xl&unscoped_q=%24spacing-2xl

It's also used by a few Protocol components: https://github.com/mozilla/protocol/search?q=%24spacing-2xl&unscoped_q=%24spacing-2xl

I'm not saying we shouldn't remove it, but we should probably try and justify it first, as updating all the places where this is used will be time consuming. What's the rationale for removing it?